datastore/src/main/java/io/spine/server/storage/datastore/DsProperties.java

```diff
@@ -67,10 +94,58 @@ static boolean isDeleted(Entity entity) {
         return hasFlag(entity, LifecycleFlagField.deleted.toString());
     }
 
+    static OrderBy byCreatedTime() {
+        return AggregateEventRecordOrdering.BY_CREATED.getOrdering();
+    }
+
+    static OrderBy byVersion() {
+        return AggregateEventRecordOrdering.BY_VERSION.getOrdering();
+    }
+
+    static OrderBy byRecordType() {
+        return AggregateEventRecordOrdering.BY_SNAPSHOT.getOrdering();
+    }
+
     private static boolean hasFlag(Entity entity, String flagName) {
         final boolean result =
                 entity.contains(flagName)
                         && entity.getBoolean(flagName);
         return result;
     }
+
+    private enum AggregateEventRecordProperty implements StorageField {
+
+        /**
+         * A property storing the Event creation time.
+         */
+        created,
```

Comment:
As commented above, let's standardize on uppercase and `toString()` that would convert to lowercase. In Java 8 it would be default method.